### PR TITLE
Add otherProps to datePicker

### PR DESCRIPTION
**Pull request by JohnnP**
### Description

Add otherProps in DatePicker components props to be able to use more feature of react-date-picker

### CODE REVIEW

**Comment by GuenoleK** [comment]
@JohnnP could you check this PR please. Test didn't pass for this one

**Comment by TomGallon** [comment]
@JohnnP thanks for you PR.

But I cannot merge your PR for this reason:
- i dont want to add otherProps cause it will display a lot of react warning for each props potentially not declared properly
- i want functionnalities of the components to be explicit for all users of the library.

if you need min and max values, please declare it explicitely

File: src/components/input/date/index.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ class InputDate extends Component {
     };
 
     render() {
-        const {error, locale, name, placeholder, disabled} = this.props;
+        const {error, locale, name, placeholder, disabled, ...otherProps} = this.props;
```
Comment (TomGallon) [inline src/components/input/date/index.js]: Please declare the props explicitely

**Comment by TomGallon** [comment]
@JohnnP 
ah ! and the tests failed !
before submit your PR, you can run `npm run test` to test it.
And please update the tests !

**Comment by JohnnP** [comment]
I changed to declare the props explicitely. And I run the tests. They passed

**Pull request merged**